test/runtests.jl

```diff
@@ -597,10 +612,6 @@ end
 
 @test sprint(showerror, AmosException(1)) == "AmosException with id 1: input error."
 
-# Used to check method existence below
-struct NotAFloat <: AbstractFloat
-end
-
```

Comment:
any reason why this was removed?

Reply:
No. That was by mistake .. Added back.
Thanks for pointing out.